src/ios/AppDelegate+FCMPlugin.m

```diff
@@ -46,7 +46,7 @@ - (BOOL)application:(UIApplication *)application customDidFinishLaunchingWithOpt
     }
 
     // [START configure_firebase]
-    [FIRApp configure];
+    //[FIRApp configure];
```

Comment:
Sorry, I was using this in conjunction with the firebase plugin before so the app crashes when they both try and configure. I can undo and resubmit the pull or however you'd like to handle it.